src/data/config_flow.ts

```diff
@@ -4,6 +4,8 @@ import { debounce } from "../common/util/debounce";
 import { getCollection, Connection } from "home-assistant-js-websocket";
 import { LocalizeFunc } from "../common/translations/localize";
 
+export const DISCOVERY_SOURCES = ["homekit", "ssdp"];
```

Comment:
Devices that are handled by the `homekit_controller` integration (and responsible for most of the 'heat' in https://github.com/home-assistant/architecture/issues/250) will be for the `zeroconf` discovery source. Does that need adding here?

Reply:
Yes.